harvester/CKANRepository.py

```diff
@@ -29,8 +29,22 @@ def _crawl(self):
             "repo_refresh_days": self.repo_refresh_days, "homepage_url": self.homepage_url
         }
         self.repository_id = self.db.update_repo(**kwargs)
+
         records = self.ckanrepo.action.package_list()
 
+        # If response is limited to 1000, get all records with pagination
+        if len(records) == 1000:
```

Comment:
Is there no way to tell from the response whether it got page limited? Right now 1000 is arbitrary, and if BCDC changes to say 500, this won't pick it up

Reply:
Unfortunately, not as far as I'm aware. I don't see any difference when comparing BC Data Catalogue with the Province of Alberta's reponses:
https://catalogue.data.gov.bc.ca/api/action/package_list
https://open.alberta.ca/api/action/package_list

The limit of 1000 isn't entirely arbitrary, as it's the main limit the CKAN API tends to use--although this is for other actions than package_list:
https://docs.ckan.org/en/2.8/api/index.html. I'm not sure if 

I added the if statement when I tested larger repositories that _don't_ limit their responses, because in those cases it slows it down a lot to limit to 1000 records (especially with 80+ calls for Open Data Canada). We'll still get everything if we remove the conditional.